wiki/cpp/DEVELOPER_GUIDE.md

```diff
@@ -151,16 +151,18 @@ cuML relies on `clang-format` to enforce code style across all C++ and CUDA sour
 The reasons behind these deviations from the Google style guide are given in comments [here](../../cpp/.clang-format).
 
 ### How is the check done?
-[run-clang-format.py](../../cpp/scripts/run-clang-format.py) is run first by `make`. This script runs clang-format only on modified files. An error is raised if the code diverges from the format suggested by clang-format, and the build fails.
+All formatting checks are done by this python script: [run-clang-format.py](../../cpp/scripts/run-clang-format.py) which is effectively a wrapper over `clang-format`. An error is raised if the code diverges from the format suggested by clang-format. It is expected that the developers run this script to detect and fix formatting violations before creating PR.
+
+After [PR 1468](https://github.com/rapidsai/cuml/pull/1468) goes through, [run-clang-format.py](../../cpp/scripts/run-clang-format.py) will be executed as part of our `gpuCI/cuml/style-check` CI test. If there are any formatting violations, PR author is expected to fix those to get CI passing. Steps needed to fix the formatting violations are described in the following sub-section.
```

Comment:
Tiny point, but I wouldn't mention "After PR 1468 goes through" because people won't see this doc until the PR has gone through ;)
(Not critical to change before push though)

Reply:
lol.. good point! Let me fix this now itself. Thanks.